modules/reference/pages/rpk/rpk-cluster/rpk-cluster-storage-unmount.adoc

```diff
@@ -0,0 +1,52 @@
+= rpk cluster storage unmount
+
+Unmount a topic from the Redpanda cluster and secure it in glossterm:tiered-storage[].
+
+This command performs an operation that:
+
+1. Rejects all writes to the topic.
```

Comment:
is order important here? If not, consider using bullets instead.

Reply:
I believe it is. @kbatuigas @Lazin thoughts?

Reply:
Confirmed as the order is important. Keeping the numbered list.